AsyncSwift/Observed/EventDetailView+Observed.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  EventDetailView+Observed.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/09/09.
+//
+
+import Foundation
+import EventKit
+
+extension EventDetailView {
+    final class Observed: ObservableObject {
+
+        @Published var showSheet = false
+        @Published var showingAlert = false
```

Comment:
```suggestion
        @Published var isNeededToShowSheet = false
        @Published var isShowingAlert = false
```

Bool은 Bool스러운 작명이 좋아보입니다 ~